src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -352,10 +353,10 @@ export function MakeEnrichmentsTabUI(
         },
         render:()=>{
             const store = getStore();
-            if (store.activeGroups.result!.length !== 2) {
+            if (store.activeGroups.result!.length !== 2 && !(store.activeGroups.result!.length > 2 && multiGroupAnalysesPossible)) {
```

Comment:
personally I find this logic hard to read. would be nicer if expanded more. eg
(!multiGroupAnalysisPossible && activeGroups !== 2) || (multiGroupAnalysisPossible && activeGroups < 2)